synapse/config/redis.py

```diff
@@ -34,3 +34,8 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
         self.redis_host = redis_config.get("host", "localhost")
         self.redis_port = redis_config.get("port", 6379)
         self.redis_password = redis_config.get("password")
+        self.sentinel_enabled = redis_config.get("sentinel", False)
+        self.service_name = redis_config.get("service_name")
+        self.sentinels = []
+        for sentinel in redis_config.get("sentinels", []):
```

Comment:
Because sentinels is array, I have a feeling that this and bellow line could be done as a single-liner.